src/MetadataReader.js

```diff
@@ -156,7 +164,7 @@ export class MetadataReader {
 
   get signingCert () {
     try {
-      return this.signingCerts[0].replace(/[\r\n\t\s]/gm, '')
```

Comment:
Why this change?

Reply:
`get signingCerts()` and `get encryptionCerts()` return an array that has already been normalised with `certs.map((node) => node.firstChild.data.replace(/[\r\n\t\s]/gm, ''))` so this replace seems to be redundant and not serve a purpose. I confirmed that by making sure these tests pass: https://github.com/compwright/passport-saml-metadata/blob/master/src/MetadataReader.test.js#L69